nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/tools/renderer/TextRenderer.java

```diff
@@ -28,9 +28,14 @@
 
     @Override
     public void accept(LogRecord<Record> record) {
-        Record rec = record.message();
-        System.out.println(String.format("|%s|%s|%s|%d|%s|%s|", record.offset(), rec.key,
-                watermarkString(rec.watermark), rec.data.length, rec.flags, binaryString(rec.data)));
+        try {
+            Record rec = record.message();
+            System.out.println(String.format("|%s|%s|%s|%d|%s|%s|", record.offset(), rec.key,
+                    watermarkString(rec.watermark), rec.data.length, rec.flags, binaryString(rec.data)));
+        } catch (ClassCastException e) {
+            System.out.println(
```

Comment:
Could you add a comment explaining why we may have this exception and we ignore it?